impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -339,7 +342,7 @@ public void cacheSerializingCopiers() throws Exception {
     CacheConfiguration<Long, Person> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder()
         .withResourcePools(ResourcePoolsBuilder.newResourcePoolsBuilder().heap(10, EntryUnit.ENTRIES).build())
         .add(new DefaultCopierConfiguration<Long>((Class)SerializingCopier.class,   //<1>
```

Comment:
`DefaultCopierConfiguration<Person>` right??? Since you changed the copier config type.